ksp_plugin/vessel.hpp

```diff
@@ -144,11 +146,15 @@ class Vessel {
   // Deletes the |flight_plan_|.  Performs no action unless |has_flight_plan()|.
   virtual void DeleteFlightPlan();
 
-  // Tries to extend the prediction up to and including |last_time|.  May not be
-  // able to do it next to a singularity.
-  virtual void FlowPrediction(Instant const& last_time);
+  // Tries to extend the prediction by extending the ephemeris by at most
+  // |max_ephemeris_steps_per_frame|.  No guarantees regarding the end time of
+  // the prediction when this call returns.
```

Comment:
The first sentence of this comment has little in common with the behaviour.

This doesn't really `Flow` anything directly; `RefreshPrediction`?

The parameterized one below does flow (after refreshing), so the identifier makes more sense there.

Reply:
Done.